mir_connector/inorbit_mir_connector/src/connector.py

```diff
@@ -279,6 +279,10 @@ def start(self):
             except Exception:
                 self.logger.exception("Error reporting mission")
 
+    def inorbit_connected(self):
+        """Check if the InOrbit MQTT session is connected."""
+        return self.inorbit_sess.client.is_connected()
```

Comment:
Not a fan of accessing `RobotSession.client` externally but can't think of another way. To keep in mind for future development of InOrbit Edge.